kiliautoml/utils/download_assets.py

```diff
@@ -29,29 +31,35 @@ class DownloadedText:
     content: str
 
 
+ONE_MINUTE = 60
+
+
+@limits(calls=250, period=ONE_MINUTE)
+def throttled_request(api_key, asset_content) -> Response:  # type: ignore
```

Comment:
there won't be any response if the connection fails. Which means that downstream `response.content` will throw an Exception.

Reply:
Note: this is not in the scope of this PR btw, since this was already broken before.